drivers/i3c/i3c_cdns.c

```diff
@@ -2387,6 +2387,54 @@ static void cdns_i3c_target_ibi_hj_complete(const struct device *dev)
 }
 #endif
 
+static void cdns_i3c_slv_sdr_tx_th_int_handler(const struct device *dev,
```

Comment:
https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#rule-a-2-inclusive-language

avoid using `slv`

Reply:
addressed review comment, changed slv to target